assets/js/src/tracking.js

```diff
@@ -91,70 +77,55 @@ export const trackChangeCartItemQuantity = ( { product, quantity = 1 } ) => {
 };
 
 /**
- * Track a begin_checkout and checkout_progress event
- * Notice calling this will set the current checkout step as the step provided in the parameter.
+ * Track begin_checkout event
  *
- * @param {number} step The checkout step for to track
- * @return {(function( { storeCart: Object } ): void)} A callable receiving the cart to track the checkout event.
+ * @param {Object} params The function params
+ * @param {Object} params.storeCart The cart object
  */
-export const trackCheckoutStep =
-	( step ) =>
-	( { storeCart } ) => {
-		if ( currentStep === step ) {
-			return;
-		}
-
-		trackEvent( step === 0 ? 'begin_checkout' : 'checkout_progress', {
-			items: storeCart.cartItems.map( getProductFieldObject ),
-			coupon: storeCart.cartCoupons[ 0 ]?.code || '',
-			currency: storeCart.cartTotals.currency_code,
-			value: formatPrice(
-				storeCart.cartTotals.total_price,
-				storeCart.cartTotals.currency_minor_unit
-			),
-			checkout_step: step,
-		} );
-
-		currentStep = step;
-	};
+export const trackBeginCheckout = ( { storeCart } ) => {
+	trackEvent( 'begin_checkout', {
+		currency: storeCart.totals.currency_code,
+		value: formatPrice(
+			storeCart.totals.total_price,
+			storeCart.totals.currency_minor_unit
+		),
```

Comment:
Documentation [states](https://developers.google.com/analytics/devguides/collection/ga4/reference/events?client_type=gtag#parameters_5) it should be a `number`, we send a `string`.

Reply:
Nice catch! Updated in 4a06f52